config.tmpl

```diff
@@ -9,14 +9,14 @@ USE_JETTY=0
 
 if [ "$USE_JETTY" -eq 1 ]; then
   # Admin user
-  AUTH_USER="admin1"
-  AUTH_PASS="password3"
+  AUTH_USER="fedoraAdmin"
```

Comment:
This seems like a different test if using `fedoraAdmin`. The `admin1` goes through a different logic flow.

Reply:
Yes, my understanding was that `AUTH_USER` was a user that could create objects in the repository as previously we didn't have WebAC one all the time. So like the non-Jetty version we used a `fedoraAdmin` roled user. If you wanted to test multiple levels of authz then you use `AUTH_USER2` and `AUTH_USER3`.

Now that WebAC is always on, we can change this. But the main user was considered to be an admin with access because I was worried about how the system reacted and not how the authz was interpreted.